.github/workflows/auto-publish-docker-dev-branch.yml

```diff
@@ -0,0 +1,36 @@
+name: Upload Docker Image of Latest Developer Branch to GHCR
+
+on:
+  schedule:
+    - cron: "0 16 * * *"  # Daily at noon EST
+  workflow_dispatch:
+  push:
```

Comment:
```suggestion
```
This was just to get the ball rolling (can't do manual triggers via `workflow_dispatch` until it runs at least once)